sql/src/test/java/io/crate/metadata/table/TestingTableInfo.java

```diff
@@ -281,6 +304,72 @@ public Builder addPartitions(String... partitionNames) {
             return this;
         }
 
+        public Builder addIndex(String name, List<String> columnNames, @Nullable String analyzerName) {
+            return addIndex(name, columnNames, analyzerName, Reference.IndexType.ANALYZED);
+        }
+
+        public Builder addIndex(String name, List<String> columnNames, @Nullable String analyzerName, Reference.IndexType indexType) {
```

Comment:
Why are there so many extensions necessary here? They seem a bit unrelated.

(I also think we should move away from this `TestingTableInfo` stuff, rather than extending it. )